features/search_companies.feature

```diff
@@ -0,0 +1,170 @@
+Feature: Search Companies
+
+As an SHF user
```

Comment:
Should this be "As a Visitor" ?  
Since we've been using 'visitor' to mean someone that comes to the SHF site to look at the information, but isn't logged in.  (They may or may not be a user or member, but they're not using it for those reasons at the moment.)

And we've been using "User" to specifically mean someone that has log-in credentials (because they are applying for a membership).